packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -37,7 +37,7 @@
                         button-type="secondary"
                         button-size="xsmall"
                         data-test-id="close-modal"
-                        @click.native="close">
+                        @click="close">
```

Comment:
Any particular reason for this change?

Reply:
Yes, from [v0.6.0 ](https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-button/CHANGELOG.md#v060) there is no need to add `.native` to button events

Reply:
Nice.

Reply:
@dandel10n I'm using f-button@1.1.1 and one of my tests requires `.native` still.

Reply:
Yeah, I noticed this in f-registration, investigating at the moment. I think I will remove this change from this PR and add it later when I find the answer

Reply:
@xander-marjoram @DamianMullins So I found that button works without `.native` but jest tests which use `trigger('click')` don't work as [trigger works only with native elements](https://vue-test-utils.vuejs.org/api/wrapper/trigger.html). The data-test-id ends up on a native button at the end but as we add it to f-button, jest doesn't recognise it as a native button. I will play around to see how to fix this issue but for now I reverted back `.native`.